tests/test1_login_test.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+from signUpDriver import signUp
+from loginDriver import logIn
+
+class configure:
```

Comment:
are these configure classes private to the file that they appear in? If so, start them with underscore. So along with capitalized classes, it would be _Configure

Reply:
No longer apart of the test